app/components/hiring_staff/vacancies_component.rb

```diff
@@ -64,4 +78,18 @@ def set_filters(vacancies, filters)
 
     vacancies.in_school_ids(filters[:managed_school_ids]).or(vacancies.in_central_office)
   end
+
+  def set_managed_school_ids
+    @filters[:managed_school_ids] ||= []
+    @filters[:managed_school_ids].push('school_group') if @filters[:managed_organisations]&.include?('school_group')
```

Comment:
any value in making sure this array doesn't contain anything twice, with Array.uniq or by using a Set rather than an Array?